README.md

```diff
@@ -8,3 +8,37 @@ Assemble the charts:
 [Example of minimal config](./tests/recommend.yaml)
 
 [Mutualize project](https://github.com/camptocamp/helm-mutualized/)
+
+## Mutualize
+
+### Configuration workflow
+
+Diagram of the configuration workflow:
+
+![Diagram of the configuration workflow](./mutualize.png 'Diagram of the configuration workflow')
+
+#### GitHubWebhook
+
+One `GitHubWebhook` will be create for each `mutualize.configs`.
+Those objects will be used by the
+[GitHub WebHook operator](https://github.com/camptocamp/operator-github-webhook/)
+to create the needed GitHub WebHooks.
+Those WebHooks will trigger the mutualize shared config manager on push on GitHub.
+
+#### Mutualize Shared Config Manager
+
+One `SharedConfigSource` will be create for each `mutualize.configs`.
+Those object are used by the
+[Shared Config Manager Operator](https://github.com/camptocamp/operator-shared-config-manager)
+to build a `ConfigMap` (from the `SharedConfigConfig` object) that contains the master config of the mutualize
+Shared Config Manager.
+With this config, the Shared Config Manager will checkout the configured repository,
+Do the variable replacement, and distribute the to the mutualize services.
+
+#### Mutualize TileClout-chain hosts
+
+One other `SharedConfigSource` will be create for each `config.hosts` for `config` for each `mutualize.configs`
```

Comment:
```suggestion
One other `SharedConfigSource` will be create for each entry in key `hosts` of each entry in `geomapfish.mutualize.configs` if `tilecloudchain` is `true`.
```

But I'm not sure we still need this as we will use /tiles the GMF project's ingress.

Reply:
Yes :-)